android/anc/src/main/java/org/smartregister/fhircore/anc/ui/details/PatientDetailsActivity.kt

```diff
@@ -159,11 +159,16 @@ class PatientDetailsActivity : BaseMultiLanguageActivity() {
       val patientDetails =
         listOf(this.patientDetails.name, this.patientDetails.gender, this.patientDetails.age)
           .joinToString(separator = ", ")
-      val patientId =
-        listOf(this.patientDetailsHead.demographics, this.patientDetails.patientIdentifier)
-          .joinToString(separator = " ID: ")
+      val patientAddress =
+        if (this.patientDetailsHead.demographics.isNotBlank()) this.patientDetailsHead.demographics
+        else this.patientDetails.demographics
+      val houseHoldHeadText =
+        if (this.patientDetails.isHouseHoldHead) getString(R.string.head_of_household) else ""
+      val patientIdText =
+        listOf(patientAddress, "ID: " + this.patientDetails.patientIdentifier, houseHoldHeadText)
```

Comment:
You can refactor the string concatenation to use string interpolation.
`"ID: ${this.patientDetails.patientIdentifier}"`